### ISSUE TITLE
[FEATURE] Automtically suscribe users to tags determined by administrators so they receive the bulletin

### ISSUE THREAD

**Issue by albjeremias** [helpbuttons/helpbuttons#810]
have an option for admins to add tags which users automatically follow when signing up, so they will received notifications of such buttons!

**Comment by albjeremias** [comment]
I disagree on having buttons with "no location" that breaks all the sense of helpbuttons, i dont think is a good idea.

and the "AdminAlert" seems very complicated...

**Comment by watchoutfreedom** [comment]
Forget about no location, it would be more like "area location" . It's one of the options considered in penpot menu from the beggining. It sound like a different buttonTemplatefeature too

**Comment by watchoutfreedom** [comment]
So admins announcement will create a button that has presence in all the network, and we can develop the area logic for this and from the rest of area hepbuttons

**Comment by albjeremias** [comment]
> So admins announcement will create a button that has presence in all the network area, and we can develop the area logic for this and from the rest of area hepbuttons

can we keep annoucements as only annoucements ? I find it so complex to mix it up with buttons which have a very specific behavior... can be just a banner on the home info.

**Comment by watchoutfreedom** [comment]
Ok, makes sense

**Comment by watchoutfreedom** [comment]
@albjeremias special features for the admins:

- The original admin, creator of the app, can be protected from being erased or excluded from the app.
- Admins have an option to send announcements to all users in the app. That would be in Network settings, ideally under moderation. The Announcements would appear under Activity in the home and on the Users activity feed, it would be notified by email if it doesn't create an issue with the mail provider.

**Comment by albjeremias** [comment]
- add option to button to pin/unpin it only for admins
